src/elements/ApplicationStore/ApplicationStore.tsx

```diff
@@ -1,46 +1,51 @@
-import React from "react";
-import styles from "./ApplicationStore.module.sass";
-import Image from "next/image";
+import Image from 'next/image'
+import React from 'react'
+
+import styles from './ApplicationStore.module.sass'
 
 type appStoreProp = {
-  appClass?: string;
-};
+  appClass?: string
```

Comment:
`className` as usual it's className or `ContainerClassName` depends on uasage place and semantic of the props